### PR TITLE
Run test-data as a part of CI

### PR DESCRIPTION
**Pull request by oyvindeide**
Blocked by https://github.com/equinor/ert/pull/1316

### CODE REVIEW

**Comment by markusdregi** [comment]
Very good initiative 👏 

However, they are not really examples, but test data... Perhaps we can call the pipeline something like `run_ert2_test_data_setups` instead?

File: .github/workflows/run_ert2_test_data_setups.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Python package
```
Comment (markusdregi) [inline .github/workflows/run_ert2_test_data_setups.yml:1]: Should probably be something along the lines of the filename...

File: .github/workflows/run_ert2_test_data_setups.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
```
Comment (markusdregi) [inline .github/workflows/run_ert2_test_data_setups.yml:13]: Should perhaps add `3.6` as that is out target Python release internally at the moment?

**Comment by markusdregi** [comment]
Besides the minor comments and the failing tests this is looking really good 👌

File: .github/workflows/run_ert2_test_data_setups.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.6]
```
Comment (markusdregi) [inline .github/workflows/run_ert2_test_data_setups.yml:13]: The order of the versions is itching me. My apologies, but can you please fix it 😂

**Review by markusdregi — APPROVED** [review]
Great initiative! 👏 I hope you fix the comment and merge it afterwards...

File: .github/workflows/run_ert2_test_data_setups.yml (hunk 3 above)
Comment (oyvindeide) [reply .github/workflows/run_ert2_test_data_setups.yml]: Not a fan of appending? ;) Fixed!

**Pull request merged**